ios/l3_interfaces/deleted_example_01.txt

```diff
@@ -0,0 +1,52 @@
+# Using Deleted
+
+# Before state:
+# -------------
+#
+# vios#show running-config | section ^interface
+# interface GigabitEthernet0/1
+#  ip address 192.0.2.10 255.255.255.0
+#  shutdown
+#  duplex auto
+#  speed auto
+# interface GigabitEthernet0/2
+#  description Configured by Ansible Network
+#  ip address 192.168.1.0 255.255.255.0
+# interface GigabitEthernet0/3
+#  description Configured by Ansible Network
+#  ip address 192.168.0.1 255.255.255.0
+#  shutdown
+#  duplex full
+#  speed 10
+#  ipv6 address FD5D:12C9:2201:1::1/64
+# interface GigabitEthernet0/3.100
+#  encapsulation dot1Q 20
```

Comment:
`encapsulation` should be handled in the l2_interfaces module IMHO

Reply:
@cidrblock `trunk encapsulation` is handled in `l2_interfaces`, but I have included the support for `l3_interfaces encapsulation` coz without setting vlan be it dot1q/isl we cannot set the sub-interface ip address.